src/abstractdataframe/selection.jl

```diff
@@ -1726,34 +1787,36 @@ function manipulate(dfv::SubDataFrame, @nospecialize(args...); copycols::Bool, k
 end
 
 manipulate(df::DataFrame, args::AbstractVector{Int}; copycols::Bool, keeprows::Bool,
-           renamecols::Bool) =
+           renamecols::Bool, multithreaded::Bool) =
     DataFrame(_columns(df)[args], Index(_names(df)[args]), copycols=copycols)
 
 function manipulate(df::DataFrame, c::MultiColumnIndex; copycols::Bool, keeprows::Bool,
-                    renamecols::Bool)
+                    renamecols::Bool, multithreaded::Bool)
     if c isa AbstractVector{<:Pair}
         return manipulate(df, c..., copycols=copycols, keeprows=keeprows,
-                          renamecols=renamecols)
+                          renamecols=renamecols, multithreaded=multithreaded)
     else
         return manipulate(df, index(df)[c], copycols=copycols, keeprows=keeprows,
-                          renamecols=renamecols)
+                          renamecols=renamecols, multithreaded=multithreaded)
     end
 end
 
 function manipulate(dfv::SubDataFrame, args::MultiColumnIndex;
-                    copycols::Bool, keeprows::Bool, renamecols::Bool)
+                    copycols::Bool, keeprows::Bool, renamecols::Bool, multithreaded::Bool)
     if args isa AbstractVector{<:Pair}
         return manipulate(dfv, args..., copycols=copycols, keeprows=keeprows,
-                          renamecols=renamecols)
+                          renamecols=renamecols, multithreaded=multithreaded)
     else
         return copycols ? dfv[:, args] : view(dfv, :, args)
     end
 end
 
 manipulate(df::DataFrame, c::ColumnIndex; copycols::Bool, keeprows::Bool,
-           renamecols::Bool) =
-    manipulate(df, Int[index(df)[c]], copycols=copycols, keeprows=keeprows, renamecols=renamecols)
+           renamecols::Bool, multithreaded::Bool) =
+    manipulate(df, Int[index(df)[c]], copycols=copycols, keeprows=keeprows,
+               renamecols=renamecols, multithreaded=multithreaded)
 
 manipulate(dfv::SubDataFrame, c::ColumnIndex; copycols::Bool, keeprows::Bool,
-           renamecols::Bool) =
-    manipulate(dfv, Int[index(dfv)[c]], copycols=copycols, keeprows=keeprows, renamecols=renamecols)
+           renamecols::Bool, multithreaded::Bool) =
```

Comment:
Why have you decided to pass `mulithreaded` to `manipulate` and `_manipulate`? It is ignored there now (it might not be in the future so it is not a problem - I just wanted to know your rationale).